pkg/local_object_storage/metabase/lock.go

```diff
@@ -0,0 +1,159 @@
+package meta
+
+import (
+	"bytes"
+	"fmt"
+
+	apistatus "github.com/nspcc-dev/neofs-sdk-go/client/status"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"go.etcd.io/bbolt"
+)
+
+// bucket name for locked objects.
+var bucketNameLocked = []byte(invalidBase58String + "Locked")
+
+// suffix for container buckets with objects of type LOCK.
+const bucketNameSuffixLockers = invalidBase58String + "LOCKER"
+
+// returns name of the bucket with objects of type LOCK for specified container.
+func bucketNameLockers(idCnr cid.ID) []byte {
+	return []byte(idCnr.String() + bucketNameSuffixLockers)
+}
+
+// Lock marks objects as locked with another object. All objects from the
```

Comment:
did not get the second sentence here

Reply:
`object from the specified container` or what? Seems pretty obvious, isn't it?

Reply:
`All objects from the...` maybe "must" word is missing?

Reply:
Added `are`.